Gemfile

```diff
@@ -38,6 +38,7 @@ gem "dalli",                          "=2.7.6",        :require => false
 gem "default_value_for",              "~>3.0.3"
 gem "docker-api",                     "~>1.33.6",      :require => false
 gem "elif",                           "=0.1.0",        :require => false
+gem "et-orbi"
```

Comment:
Can you mention that this is temporary until/if we can get a patch accepted upstream?  Is this relevant for all versions of et-orbi?

Reply:
Yeuppp.